### ISSUE TITLE
rename to mapeo-specs

### ISSUE THREAD

**Issue by okdistribute** [digidem/mapeo-rfc#2]
'rfc' is fine but specs is a more clear name

**Comment by gmaclennan** [comment]
_if_ that is what we want to limit this too. I had been thinking that this was a place to share ideas about how to implement a particular feature and get feedback, particularly features that don't clearly belong in a particular module issues. I'm happy to keep this to specs, as long as we agree where to put other discussions, e.g. we come up with an idea about how to implement user access control, and want to share and collect comments, where should it go?

**Comment by okdistribute** [comment]
I think that can also be here, and then when that document is merged it still goes here as a reference for how it works.. open to other ideas though

**Comment by hackergrrl** [comment]
I agree with @gmaclennan's description of what the repo is for. I'm ok with any of

- `mapeo-discuss`
- `mapeo-rfc`
- `mapeo-specs`
- `mapeo-tech-design`

**Comment by okdistribute** [comment]
From an outsider's viewpoint as well as a technology grant viewpoint, it would be nice to have some way to see the specs or tech-design. I guess 'rfc' can also indicate this, but 'discuss' definitely feels a bit informal for the level of detail we are reaching with these documents! (which is great!)
